### PR TITLE
init: connecting MUI and react router dom

### PR DESCRIPTION
**Pull request by Farinoxa**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,10 @@
 # testing
 /coverage
 
+# next.js
+/.next/
+/out/
+
```
Comment (des1nteresado) [inline .gitignore]: Please remove unused rules.

File: .gitignore [hunk 2]
```diff
@@ -17,7 +21,9 @@
 .env.development.local
 .env.test.local
 .env.production.local
+*.pem
```
Comment (des1nteresado) [inline .gitignore:22]: Please add missed .vscode

File: .gitignore (hunk 1 above)
Comment (Farinoxa) [reply .gitignore]: unused rules were removed

File: .gitignore (hunk 2 above)
Comment (Farinoxa) [reply .gitignore:22]: added .vscode

**Pull request merged**